streams/readable-streams/tee.any.js

```diff
@@ -317,6 +317,60 @@ promise_test(t => {
 
 }, 'ReadableStream teeing: erroring the original should immediately error the branches');
 
+promise_test(t => {
```

Comment:
Could use async/await to ease readability

Reply:
The original streams tests were written without `async`/`await` but I support using it going forward.

Reply:
I went with promise chaining for consistency with the other tests in the same file. But if you're okay with writing new tests with `async`/`await`, then I'll quickly rework them. (Maybe we can refactor the existing tests in a separate PR? 😁)